packages/jseql/src/eql.ts

```diff
@@ -0,0 +1,98 @@
+const { newClient, encrypt, decrypt } = require('@cipherstash/jseql-ffi');
```

Comment:
`import` syntax doesn't seem to work (likely a bug in '@cipherstash/jseql-ffi'). Haven't debugged the issue yet, but using `require` works.

Reply:
Also haven't added type defs for anything in `@cipherstash/jseql-ffi` yet.